src/Tizen.NUI/src/public/Visuals/VisualMaps.cs

```diff
@@ -82,6 +82,25 @@ public class VisualMap : IDisposable
         [EditorBrowsable(EditorBrowsableState.Never)]
         private Vector4 cornerRadius = null;
 
+        /// <summary>
+        /// The borderline width of the visual.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        private float? borderlineWidth = null;
```

Comment:
Why do you declare it? I think you no need it, because you use `BindableProperty`, it used for property bag.

Reply:
it is my miss understand, this class is not View and ViewStyle.